src/main/java/org/commonjava/indy/service/tracking/client/content/ContentService.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (C) 2023 Red Hat, Inc. (https://github.com/Commonjava/indy-tracking-service)
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.service.tracking.client.content;
+
+import org.commonjava.indy.service.tracking.client.CustomClientRequestFilter;
+import org.commonjava.indy.service.tracking.model.TrackedContentEntry;
+import org.eclipse.microprofile.rest.client.annotation.RegisterProvider;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+import java.util.Set;
+
+@Path( "/api/admin/content" )
+@RegisterRestClient( configKey = "content-service-api" )
+@RegisterProvider( CustomClientRequestFilter.class )
+public interface ContentService
+{
+    @POST
+    @Path( "/tracking/recalculate" )
+    Response recalculateEntrySet( final Set<TrackedContentEntry> entries );
```

Comment:
I'm thinking if we should use a cut version of this TrackedContentEntry to reduce the payload size. From code I see that the recalculate only needs these fields: trackingKey, storeKey, path, access-channel, effect, originalUrl. So maybe the cut one should be like:
{ 
trackingKey: "xxx",
entries: [
{ storeKey: "xxxx", path: "xxx", accesschannel: "xxx", originalUrl: "xxx", effect: "xxx"}, .....
]
}

Reply:
I am still a little bit confused about how to create a cut version of this but I have given it a try. Please take a look at it.

Reply:
If you check the code here: https://github.com/Commonjava/indy/blob/3.1.x/addons/folo/common/src/main/java/org/commonjava/indy/folo/ctl/FoloAdminController.java#L462
You will find that the folo recalculate only cares about cut version of fileds in TrackedContentEntry to do the Transfer related jobs in indy-monolith(Content service). So that's why I'm considering to use a cut version to reduce payload size between tracking-service and content-service.

Reply:
thanks. I have created it according to your suggestion.